readyplayerme/rpm-unity-sdk-core
# [SDK-374] - move webgl under core samples

## [SDK-374](https://ready-player-me.atlassian.net/browse/SDK-374)

## Description

-   Move WebGL sample under the Core package sample.






## How to Test

-   Import package to new project and check, that the import is asked and package works as expected after building



## Checklist
